### PR TITLE
Fix _renderChildren error in Safari.

### PR DESCRIPTION
**Pull request by brettdewoody**
Fixes the error in Safari due to use of arrow functions.

### CODE REVIEW

File: lib/ReactFitText.js [hunk 1]
```diff
@@ -48,8 +48,13 @@ module.exports = React.createClass({
...
   _renderChildren: function(){
-    return React.Children.map(this.props.children, child =>
-      React.cloneElement(child,{ref:(c) => this._childRef = c})
+    return React.Children.map(this.props.children, function(child) {
+      return React.cloneElement(child, {
+          ref: function(c) {
+            return this._childRef = c
+          }.bind(this)
+        })
+      }.bind(this)
```
Comment (enrtama) [inline lib/ReactFitText.js]: Sorry, the _renderChildren function should be replace by:

  _renderChildren: function _renderChildren() {
    var _this = this;

    return React.Children.map(this.props.children, function (child) {
      return React.cloneElement(child, { ref: function ref(c) {
        return _this._childRef = c;
      } });
    });
  },

I've tested it and it works.

**Comment by brettdewoody** [comment]
Thanks for catching @enrtama, when I was trying to figure it out I went the `.bind(this)` path and didn't look at the bigger picture. PR is updated.

**Comment by enrtama** [comment]
That looks good, ready to merge. @brettdewoody

**Comment by roblingle** [comment]
@gianu Can this be pushed to npm? I added tried the package and just ran into this problem.

**Pull request merged**